apps/main/src/llamalend/queries/repay/repay-expected-borrowed.query.ts

```diff
@@ -30,20 +30,21 @@ export const { useQuery: useRepayExpectedBorrowed } = queryFactory({
       { userBorrowed },
     ] as const,
   queryFn: async ({ marketId, stateCollateral, userCollateral, userBorrowed }: RepayFromCollateralQuery) => {
+    // todo: investigate if this is OK when the user's position is not leveraged
     const market = getLlamaMarket(marketId)
-    return market instanceof LendMarketTemplate
-      ? ((await market.leverage.repayExpectedBorrowed(
-          stateCollateral,
-          userCollateral,
-          userBorrowed,
-        )) as RepayExpectedBorrowedResult)
-      : market.leverageV2.hasLeverage()
-        ? ((await market.leverageV2.repayExpectedBorrowed(
-            stateCollateral,
-            userCollateral,
-            userBorrowed,
-          )) as RepayExpectedBorrowedResult)
-        : null
+    if (market instanceof LendMarketTemplate) {
+      const result = await market.leverage.repayExpectedBorrowed(stateCollateral, userCollateral, userBorrowed)
+      return result as RepayExpectedBorrowedResult
```

Comment:
I'm in favor of Typescript implicitly inferring ther return type of this query. That was you can also get rid of the manual type definition on top